data/migrations/20210802234758_storyPromptsTable.js

```diff
@@ -0,0 +1,16 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('Story-Prompts', (sp) => {
```

Comment:
I don't think your table name follows the standard on this repo for using CamelCase.

Reply:
I think it looks consistent to most other compound table names.  Looks like 1 table is showing camel case but multiple others have the hyphenated naming case.

Reply:
Ah, yes. You're right about that. I got confused with what I was looking at. This is good then. 👍🏻